testbed/src/main/java/org/spdx/testbed/TestCase.java

```diff
@@ -7,8 +7,17 @@
 import org.spdx.library.InvalidSPDXAnalysisException;
 import org.spdx.tools.InvalidFileNameException;
 
+import javax.annotation.Nonnull;
+
 import java.io.IOException;
 
-public interface TestCase {
-    TestResult test(String[] args) throws InvalidSPDXAnalysisException, IOException, InvalidFileNameException;
+public interface TestCase extends Comparable<TestCase> {
+    TestResult test(String[] args) throws InvalidSPDXAnalysisException, IOException,
```

Comment:
Hm, good question. I think we might as well make it a single string for now. Even for conversion test cases, my plan would be to execute the same test a number of times (with an additional wrapper that verifies the input format), so that should be covered by a single input as well

Reply:
Changed it